xcode/Subconscious/Shared/Components/Common/Profile/ProfilePic.swift

```diff
@@ -13,39 +13,75 @@ enum ProfilePicVariant: Equatable, Codable, Hashable {
     case image(String)
 }
 
-struct ProfilePicImage: View {
-    var image: Image
-    
-    var body: some View {
-        image
-            .resizable()
+enum ProfilePicSize {
+    case small
+    case medium
+    case large
+}
+
+fileprivate struct ProfilePicFrameViewModifier: ViewModifier {
+    let size: CGFloat
+    let lineWidth: CGFloat
+
+    func body(content: Content) -> some View {
+        content
             .aspectRatio(contentMode: .fill)
-            .frame(width: 48, height: 48)
+            .frame(width: size, height: size)
             .clipShape(Circle())
-            .overlay(Circle().stroke(Color.separator, lineWidth: 0.5))
+            .overlay(Circle().stroke(Color.separator, lineWidth: lineWidth))
+    }
+}
+
+extension View {
+    fileprivate func profilePicFrame(size: CGFloat, lineWidth: CGFloat) -> some View {
```

Comment:
Interesting! I've mostly avoided fileprivate, because it is a "fancy" scoping mechanism that might not be familiar to people who aren't used to Swift's odd project-level scoping rules.

It might not be a bad idea to make this a generally available method, and give it is own file `Common/ProfilePickframeViewModifier.swift`

Non-blocking.